### ISSUE TITLE
Auto repair finetuning

### ISSUE THREAD

**Issue by mmd-osm** [tyrasd/overpass-turbo#322]
Sometimes auto repair is a bit too eager and gets into the way of returning correct data.

- [x] Counting tags via `out count`

```
(
  node["place:importance"];
  way["place:importance"];
  relation["place:importance"];
);
out count;
```

Auto repair suggests to fix this query. However, the new query has a completely different semantic.

```C
(
  node["place:importance"];
  way["place:importance"];
  relation["place:importance"];
);

/*added by auto repair*/
(._;>;);
/*end of auto repair*/
out count;
```

Suggestion would be to turn off auto repair for `out count;`  altogether.

**Comment by tyrasd** [comment]
there's a similar (but less important) issue with the repair routine for the JOSM/Level0 export:

```
(
  node["place:importance"];
  way["place:importance"];
  relation["place:importance"];
);
out meta count;/*fixed by auto repair*/
```

**Comment by thibaultmol** [comment]
I made a fix for this.... in the sense that I modified the code so that if It notices that you're trying to count something; it'll just not show the auto repair prompt.
Is that a perfect solution? no... but this issue has been open for literally 6 years... So I think this is better than nothing.

**Comment by tyrasd** [comment]
fixed in #639 and 6215541
